R/estimate_KM.R

```diff
@@ -17,15 +14,19 @@
 #'
 #' @seealso \code{\link[survival]{survfit.formula} \link[survival]{survfitCI}}
 #'
-#' @param data The name of the dataset for Time-to-Event analysis based on the Analysis Data Model (ADaM) principles. The dataset is expected to have
```

Comment:
Should be kept. Our main focus still is data standards. Non-standard data is the exception that we are allowing for, not what we design for.

Reply:
CDISC is mention 1 million times in the documentation: there is no missing is. This argument now applies to both non-CDISC data and CDISC, and there is no value added adding it here as well. We also do not need to add even more text than there already is to the documentation in this argument explaining this

Reply:
There is added value for statistical programmers. They like things explicit.

Reply:
there are 5 references to CDISC data in this single help file. The line directly below is "default values follow the naming conventions in CDISC." This is quite explicit

Reply:
See above - I think a statement at the start then is enough throughout the rest of the help file.

Reply:
An update to the vignette - one that is not about CDISC - should be sufficient IMO.